modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -27,7 +27,7 @@ For {productname} {smr} hardware requirements, see xref:retail:retail-requiremen
 [[server-hardware-requirements]]
 == Server Requirements
 
-One of {sle-micro} {microversion} or {sles} {sles-version} {sp-version} is the operating system of the container host.
+One of {sle-micro} {microversion} or {sles} {sles-version} {bci-suma} is the operating system of the container host.
```

Comment:
If we want to go for the `bci-suma` macro, we must drop `sles-version`.  Some of us seem to prefer it:
```suggestion
One of {sle-micro} {microversion} or {sles} {bci-suma} is the operating system of the container host.
```
The disadvantage is that we must replace it with `bci-mlm` in master.

Reply:
messed it up during during changing this. {sles-version}  is wrong as it was pointing to 15SP5.